docs/timezone.md

```diff
@@ -1,19 +1,213 @@
+  Allowed values are `earlier`, `later`, and `reject`.
+  The default is `earlier`.
+
+**Returns:** A `Temporal.Absolute` object indicating the absolute time in `timeZone` at the time of the calendar date and wall-clock time from `dateTime`.
+
+This method is one way to convert a `Temporal.DateTime` to a `Temporal.Absolute`.
+
+### Resolving ambiguity
+
+> This explanation was adapted from the [moment-timezone documentation](https://github.com/moment/momentjs.com/blob/master/docs/moment-timezone/01-using-timezones/02-parsing-ambiguous-inputs.md).
+
+This conversion is not necessarily one-to-one.
+Due to DST time changes, there is a possibility that a wall-clock time either does not exist, or has existed twice.
+The `disambiguation` argument controls what absolute time to return in the case of ambiguity:
+- `earlier` (the default): The earlier of the two possible absolute times will be returned.
+- `later`: The later of the two possible absolute times will be returned.
+- `reject`: A `RangeError` will be thrown.
+
+When entering DST, clocks move forward an hour.
+In reality, it is not time that is moving, it is the offset moving.
+Moving the offset forward gives the illusion that an hour has disappeared.
+As the clock ticks, you can see it move from 1:58 to 1:59 to 3:00.
+It is easier to see what is actually happening when you include the offset.
+
+```
+1:58 +1
+1:59 +1
+3:00 +2
+3:01 +2
+```
+
+The result is that any time between 1:59:59 and 3:00:00 never actually happened.
+In `earlier` mode, the absolute time will skip backwards by the amount of the DST gap (usually 1 hour).
+In `later` mode, the absolute time will skip forwards by the amount of the DST gap.
+
+```javascript
+tz = new Temporal.TimeZone('Europe/Berlin');
+dt = new Temporal.DateTime(2019, 3, 31, 2, 45);
+tz.getAbsoluteFor(dt, 'earlier');  // => 2019-03-31T00:45Z
+tz.getAbsoluteFor(dt, 'later');    // => 2019-03-31T01:45Z
+tz.getAbsoluteFor(dt, 'reject');   // throws
+```
+
+In this example, the wall-clock time 2:45 doesn't exist, so it is treated as either 1:45 +01:00 or 3:45 +02:00.
+
+Likewise, at the end of DST, clocks move backward an hour.
+In this case, the illusion is that an hour repeats itself.
+In `earlier` mode, the absolute time will be the earlier instance of the duplicated wall-clock time.
+In `later` mode, the absolute time will be the later instance of the duplicated time.
+
+```javascript
+tz = new Temporal.TimeZone('America/Sao_Paulo');
+dt = new Temporal.DateTime(2019, 2, 16, 23, 45);
+tz.getAbsoluteFor(dt, 'earlier');  // => 2019-02-17T01:45Z
+tz.getAbsoluteFor(dt, 'later');    // => 2019-02-17T02:45Z
+tz.getAbsoluteFor(dt, 'reject');   // throws
+```
+
+In this example, the wall-clock time 23:45 exists twice.
+
+> *Compatibility Note*: The built-in behaviour of the Moment Timezone and Luxon libraries is to give the same result as `earlier` when turning the clock back, and `later` when setting the clock forward.
```

Comment:
This seems to indicate a gap in the Temporal API that should be corrected, and closely relates to [Preserving local time](https://github.com/tc39/proposal-temporal/issues/240#user-content-preserving-local-time) (cc @pdunkel).

Reply:
Opened https://github.com/tc39/proposal-temporal/issues/318 for this.